elementary/monitor/dbt_project/models/alerts/alerts_source_freshness.sql

```diff
@@ -0,0 +1,19 @@
+{{
+  config(
+    materialized = 'incremental',
+    unique_key = 'alert_id',
+    merge_update_columns = ['alert_id'],
+    on_schema_change = 'append_new_columns'
+  )
+}}
+
+
+{% set error_source_freshness_relation = adapter.get_relation(this.database, this.schema, 'alerts_dbt_source_freshness') %}
```

Comment:
cosmetics - the name here is confusing IMO because when I read it I think that this is a relation contains errors in source freshness and not alerts

Reply:
Agreed. Copied it from `alerts_models` but there the alerts are actually errors so it's different.